client/components/stars.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import propTypes from 'proptypes';
+import Star from './star';
+
+const Stars = (props) => {
+  const { rating } = props;
+  const starArray = [];
+
+  for (let i = 0; i < 5; i += 1) {
+    if (i < rating) {
+      starArray.push('selectedStar');
+    } else {
+      starArray.push('greyStar');
+    }
+  }
+
+  return (
+    <div className="starBox">
+      {starArray.map(star => (
+        <Star type={star} />
```

Comment:
It may be more clear here if you push a star component with the class you're expecting into the starArray and then return the star Array with all stars in this div here. Also less iteration. Looks good though and a minor comment. I see that's the only thing you did differently than me.

Reply:
I did it this way because if I added the class to the Star while invoking it inside the Stars component, my tests wouldn't register that class. But they registered the class if I gave the star the class from within the Star component. This way I was able to test that the right classes were being assigned. But you're right, that way would also be functional :)